README.md

```diff
@@ -1,34 +1,251 @@
+- **[linters]**
+- **[github]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://github.com/gonzaloalonzo/hellomicroverse)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> You may fork or download the proyect
+
+### Prerequisites
+
+In order to run this project you need:
+
+Browser for internet
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/gonzaloalonzo/hellomicroverse
+```
+
+
+### Install
+
+Install this project with:
+
+    sh
+  cd my-project
+  gem install https://github.com/gonzaloalonzo/hellomicroverse
+
+
+### Usage
+
+To run the project, execute the following command:
+
+https://github.com/gonzaloalonzo/hellomicroverse
+
+### Run tests
+
+To run tests, run the following command:
+
+    sh
+  bin/rails test test/models/article_test.rb
+
+
+### Deployment
+
+You can deploy this project using:
+
+https://github.com/gonzaloalonzo/hellomicroverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+Gonzalo alonzo, Julio Cezar
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/gonzaloalonzo/hellomicroverse)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[images]**
+- [ ] **[Video]**
+- [ ] **[more text]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> It was difficul but effort keep me going thank to all in microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Thank to my mother(RIP).
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> How to conect css?
+
+
+- **[Question_1]**
+
+  - [use the link tag]
```

Comment:
- [OPTIONAL] You have an answer to a question that is yet to be asked 😆. It would be nice to add a question there. 👍